cime_config/machines/config_machines.xml

```diff
@@ -2850,73 +2849,72 @@
       </arguments>
     </mpirun>
     <module_system type="module" allow_error="true">
-      <init_path lang="sh">/sw/summit/lmod/7.7.10/rhel7.3_gnu4.8.5/lmod/lmod/init/sh</init_path>
-      <init_path lang="csh">/sw/summit/lmod/7.7.10/rhel7.3_gnu4.8.5/lmod/lmod/init/csh</init_path>
-      <init_path lang="python">/sw/summit/lmod/7.7.10/rhel7.3_gnu4.8.5/lmod/lmod/init/env_modules_python.py</init_path>
-      <init_path lang="perl">/sw/summit/lmod/7.7.10/rhel7.3_gnu4.8.5/lmod/lmod/init/perl</init_path>
+      <init_path lang="sh">/sw/summit/lmod/8.4/init/sh</init_path>
+      <init_path lang="csh">/sw/summit/lmod/8.4/init/csh</init_path>
+      <init_path lang="python">/sw/summit/lmod/8.4/init/env_modules_python.py</init_path>
+      <init_path lang="perl">/sw/summit/lmod/8.4/init/perl</init_path>
       <cmd_path lang="perl">module</cmd_path>
-      <cmd_path lang="python">/sw/summit/lmod/7.7.10/rhel7.3_gnu4.8.5/lmod/7.7.10/libexec/lmod python</cmd_path>
+      <cmd_path lang="python">/sw/summit/lmod/8.4/libexec/lmod python</cmd_path>
       <cmd_path lang="sh">module</cmd_path>
       <cmd_path lang="csh">module</cmd_path>
       <modules>
         <command name="purge"/>
-        <command name="ls"/>
+        <command name="list"/>
         <command name="load">DefApps</command>
-        <command name="load">python/3.7.0</command>
-        <command name="load">subversion/1.9.3</command>
-        <command name="load">git/2.13.0</command>
+        <command name="load">python/3.7-anaconda3</command>
+        <command name="load">subversion/1.14.0</command>
+        <command name="load">git/2.31.1</command>
         <command name="load">cmake/3.20.2</command>
-        <command name="load">essl/6.1.0-2</command>
+        <command name="load">essl/6.3.0</command>
         <command name="load">netlib-lapack/3.8.0</command>
       </modules>
       <modules compiler="pgi.*">
-        <command name="load">pgi/19.9</command>
-        <command name="load">pgi-cxx14/default</command>
+        <command name="load">nvhpc</command>
```

Comment:
Can you add `nvhpc` version? Default versions change after PM days and loading specific version might help reproducibility.